README.md

```diff
@@ -1 +1,47 @@
 # todo-list
+The todo list application is an app built with javascript where users can remove and add tasks
+
+## Built With
+
+- Html5
+- Css3
+- Javascript
+
+## Live Demo
+[live demo](https://prolajumokeoni.github.io/todo-list/)
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+- [Copy this link](https://github.com/prolajumokeoni/todo-list)
+- Open your terminal or command line
+- Run `git clone` and Paste the link
+- Open the folder with your code editor
```

Comment:
- [ ] In this getting started part there are some instructions that you have missed for a user to use your project. such as `npm install`, and `npm start`

Reply:
- [ ] Still not resolved. Please try to follow the requirements.